<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaEquipos</class>
 <widget class="QDialog" name="VentanaEquipos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equipos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="text">
        <string>Equipos Registrados:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listaEquipos"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnActualizar">
         <property name="text">
          <string>Actualizar Lista</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminar">
         <property name="text">
          <string>Eliminar Equipo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditar">
         <property name="text">
          <string>Editar Equipo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAgregar">
         <property name="text">
          <string>Agregar Equipo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
